lib/dor/services/metadata_service.rb

```diff
@@ -69,13 +70,21 @@ def handler_for(prefix)
 
       private
 
+      # def handlers
+      #   @handlers ||= {}
+      # end
       def handlers
-        @handlers ||= {}
+        @handlers ||= {}.tap do |handlers|
+          # There's only one. If additional handlers are added, will need to be registered here.
+          register(CatalogHandler.new)
```

Comment:
I'd even get rid of the `register` logic and just return the `CatalogHandler` instance here.